lib/Extension/ClassMover/Application/ClassReferences.php

```diff
@@ -13,6 +13,15 @@
 use Phpactor\Filesystem\Domain\FilesystemRegistry;
 use Phpactor\TextDocument\TextDocumentBuilder;
 
+/** @phpstan-type ReferenceArray array{
```

Comment:
would be good to add support for this to Phpactor.

Reply:
Should I create an issue for this?

Reply:
:shrug: